frontend/src/components/SchemaEdit.vue

```diff
@@ -141,7 +115,7 @@ export default {
       toSend.add_attributes = add;
```

Comment:
I think that this needs to be 
```
const toSend = {
        name: this.details.name,
        slug: this.details.slug,
        reviewable: this.details.reviewable,
      };
```
or something like that, because currently updating these fields changes nothing

Reply:
I will need to check. I also noticed, that deleting attributes had no effect (but I also don't remember whether my schema required reviews :thinking: )

Reply:
> that deleting attributes had no effect...whether my schema required reviews

Nah, I think it was not the case😅 I somehow managed to miss this part in traceability for schema where it applies update request. So, although there is needed data to perform deletion of attribute, program just skips it🙄 It doesn't make much sense to fix this at current traceability state, because it will be rewritten anyway. But if you need this to work,
in `traceability/schema.py` in change `apply_schema_update_request()` to this
```python
def apply_schema_update_request(db: Session, change_id: int, reviewed_by: User, comment: Optional[str] = None) -> Schema:
    change = db.execute(
        select(Change)
        .where(Change.id == change_id)
        .where(Change.change_object == ChangeObject.SCHEMA)
        .where(Change.change_type == ChangeType.UPDATE)
    ).scalar()
    if change is None:
        raise MissingSchemaUpdateRequestException(obj_id=change_id)
    schema_update: SchemaUpdate = db.execute(select(SchemaUpdate).where(SchemaUpdate.change_id == change.id)).scalar()
    attrs_update: List[AttributeUpdate] = db.execute(select(AttributeUpdate).where(AttributeUpdate.change_id == change.id)).scalars().all()
    attrs_create: List[AttributeCreate] = db.execute(select(AttributeCreate).where(AttributeCreate.change_id == change.id)).scalars().all()
    attrs_delete: List[AttributeDelete] = db.execute(select(AttributeDelete).where(AttributeDelete.change_id == change.id)).scalars().all()
    data = SchemaUpdateSchema(
        name=schema_update.new_name,
        slug=schema_update.new_slug,
        reviewable=schema_update.new_reviewable,
        update_attributes=[
            AttrDefUpdateSchema(
                name=i.attribute.name,
                new_name=i.new_name,
                required=i.required,
                unique=i.unique,
                list=i.list,
                key=i.key,
                description=i.description,
                bind_to_schema=i.bind_to_schema
            )
            for i in attrs_update
        ],
        add_attributes=[
            AttrDefSchema(
                name=i.name,
                type=i.type.name,
                required=i.required,
                unique=i.unique,
                list=i.list,
                key=i.key,
                description=i.description,
                bind_to_schema=i.bind_to_schema
            )
            for i in attrs_create
        ],
        delete_attributes=[i.attribute.name for i in attrs_delete]
    )
    change.reviewed_at = datetime.utcnow()
    change.reviewed_by = reviewed_by
    change.status = ChangeStatus.APPROVED
    change.comment = comment
    s = update_schema(db=db, id_or_slug=schema_update.schema_id, data=data, commit=False)
    db.commit()
    return s
```

Reply:
OK, if you are confident that I did not break it on the frontend, it's fine for me at the moment :slightly_smiling_face:

Reply:
Well, at least the last time I checked the problem was only on backend side😏